roles/docker/vars/debian.yml

```diff
@@ -3,7 +3,6 @@ docker_kernel_min_version: '3.10'
 # https://apt.dockerproject.org/repo/dists/debian-wheezy/main/filelist
 docker_versioned_pkg:
   'latest': docker-engine
-  '1.11': docker-engine=1.11.2-0~{{ ansible_distribution_release|lower }}
```

Comment:
Actually, I retract my statement. Docker 1.11 doesn't correctly support the shared mounting. 1.12 and 1.13 do